src/elements/metadata-list/metadata-list.md

```diff
@@ -0,0 +1,20 @@
+```jsx
+  <MetadataList>
+    <MetadataList.Item label="Field of study">
+      Computer Science
+    </MetadataList.Item>
+
+    <MetadataList.Item
+      label="Publication data"
+    >
```

Comment:
```suggestion
    <MetadataList.Item label="Publication data">
```

Reply:
Maybe we should setup linter for jsx section in md files (if it is possible)